lib/kino/input.ex

```diff
@@ -257,6 +257,218 @@ defmodule Kino.Input do
     })
   end
 
+  @doc """
+  Creates a new datetime input.
+
+  ## Options
+
+    * `:default` - the initial input value. Defaults to the
+      current time
+
+    * `:min` - the minimum datetime value
+
+    * `:max` - the maximum datetime value
+
+    * `:step` - the select datetime interval
+  """
+  @spec datetime(String.t(), keyword()) :: t()
+  def datetime(label, opts \\ []) when is_binary(label) and is_list(opts) do
+    min = Keyword.get(opts, :min, nil)
+    max = Keyword.get(opts, :max, nil)
+    step = Keyword.get(opts, :step, 1)
```

Comment:
Actually, we need to be careful with step because it determines how precise is the input field and we must make sure if the step is like `60` (seconds), we don't use arbitrary value for the seconds when parsing the input value.

Honestly, I would rather skip `:step` for now, use the default step (60 for date and datetime), make sure we don't include milliseconds in the date structs and that seconds are zeroed out. If needed we can add support for seconds later, perhaps with a more strict option like `:precision`.